s3/src/main/scala/geotrellis/store/s3/S3LayerCopier.scala

```diff
@@ -42,8 +42,8 @@ class S3LayerCopier(
       val request =
         CopyObjectRequest.builder()
           .copySource(bucket + "/" + s3obj.key)
-          .bucket(bucket)
-          .key(s3obj.key.replace(s"${from.name}/${from.zoom}", s"${to.name}/${to.zoom}"))
+          .destinationBucket(bucket)
```

Comment:
That has been always a warning :/ since some sdk v2 version... I just noticed it and fixed to get rid of a deprecation here; I guess we need a separate issue to get rid of all deprecations to have an easier scala 2.13 upd